code/modules/antagonists/heretic/knowledge/cosmic_lore.dm

```diff
@@ -0,0 +1,192 @@
+		/datum/heretic_knowledge/summon/fire_shark,
+	)
+	spell_to_add = /datum/action/cooldown/spell/cosmic_rune
+	cost = 1
+	route = PATH_COSMIC
+
+/datum/heretic_knowledge/mark/cosmic_mark
+	name = "Mark of Cosmos"
+	desc = "Your Mansus Grasp now applies the Mark of Cosmos. The mark is triggered from an attack with your Cosmic Blade. \
+		When triggered, the victim is returned to the location where the mark was originally applied to them. \
+		They will then be paralyzed for 2 seconds."
+	gain_text = "As the guide was lost he found a new. The energy increased as the gaze he threw. \
+		He didn't know, but with focus, the Sleepers energy began to flow."
+	next_knowledge = list(/datum/heretic_knowledge/knowledge_ritual/cosmic)
+	route = PATH_COSMIC
+	mark_type = /datum/status_effect/eldritch/cosmic
+
+/datum/heretic_knowledge/knowledge_ritual/cosmic
+	next_knowledge = list(/datum/heretic_knowledge/spell/star_touch)
+	route = PATH_COSMIC
+
+/datum/heretic_knowledge/spell/star_touch
+	name = "Star Touch"
+	desc = "Grants you Star Touch, a spell which places a star mark upon your target \
+		and creates a cosmic field at your feet. Targets which already have a star mark \
+		will be forced to sleep for 4 seconds. When the victim is hit it also creates a beam that \
+		deals a bit of fire damage and damages the cells. \
+		The beam lasts a minute, until the beam is obstructed or until a new target has been found."
+	gain_text = "He dreamed to know how the matter travelled from star to star. \
+		He lost interest in wanting to find out."
+	next_knowledge = list(/datum/heretic_knowledge/spell/star_blast)
+	spell_to_add = /datum/action/cooldown/spell/touch/star_touch
+	cost = 1
+	route = PATH_COSMIC
+
+/datum/heretic_knowledge/spell/star_blast
+	name = "Star Blast"
+	desc = "Fires a projectile that moves very slowly and create a cosmic field on impact. \
+		Anyone hit by the projectile will recieve burn damage, a knockdown, and a star mark."
+	gain_text = "He didn't try, yet felt the call of the night's Creator."
+	next_knowledge = list(
+		/datum/heretic_knowledge/blade_upgrade/cosmic,
+		/datum/heretic_knowledge/reroll_targets,
+		/datum/heretic_knowledge/curse/corrosion,
+		/datum/heretic_knowledge/curse/cosmic_trail,
+	)
+	spell_to_add = /datum/action/cooldown/spell/pointed/projectile/star_blast
+	cost = 1
+	route = PATH_COSMIC
+
+/datum/heretic_knowledge/blade_upgrade/cosmic
+	name = "Cosmic Blade"
+	desc = "Your blade now deals damage to people's cells through cosmic radiation."
+	gain_text = "As he ascended to be a watcher, he needed to gather knowledge. \
+		He started to draw it at his home."
+	next_knowledge = list(/datum/heretic_knowledge/spell/cosmic_expansion)
+	route = PATH_COSMIC
+
+/datum/heretic_knowledge/blade_upgrade/cosmic/do_melee_effects(mob/living/source, mob/living/target, obj/item/melee/sickly_blade/blade)
+	if(source == target)
+		return
+
+	target.apply_damage_type(damage = 5, damagetype = CLONE)
```

Comment:
Clone loss is annoying to heal but this might actually be the weakest of the heretic blade upgrades, as well as just kind of... maybe a little boring? 
25% increased force is never something to be sad about mind, I just wonder if it might be more interesting if it applied the radiation debuff or something (although that has the disadvantage of not stacking).

Fine to leave it as is if you're attached to it though, I'm just musing.

Reply:
For now I'll leave it like this, if someone wants to change the ability to apply a radiation status effect with a purple glow or something they can do that.